frontend/pages/DashboardPage/DashboardPage.tsx

```diff
@@ -479,6 +479,13 @@ const DashboardPage = ({ router, location }: IDashboardProps): JSX.Element => {
       );
   };
 
+  let refetchActivities = () => {
+    /* noop */
+  };
+  const setRefetchActivities = (refetch: () => void) => {
+    refetchActivities = refetch;
+  };
+
```

Comment:
Could we use `useRef` for this? 
```
  const refetchActivitiesRef = useRef<(() => void) | null>(null);

  const setRefetchActivities = useCallback((refetch: () => void) => {
    refetchActivitiesRef.current = refetch;
  }, []);
```

Reply:
Felt like a ref wasn't quite the right thing for this as it doesn't actually need to be reactive, so the extra indirection doesn't buy us anything. Which probably means I'm missing something.